documentation/pages/move-registry/tooling/mvr-cli.mdx

```diff
@@ -0,0 +1,45 @@
+import { Tabs } from 'nextra/components'
+import { Callout } from 'nextra/components'
+
+# MVR CLI
+
+MVR CLI is used to manage your package dependencies and to interact with the Move Registry. It is a command-line tool that allows you to:
+
+1. Add dependencies to your Move project.
+2. Build your Move project with MVR dependencies.
+
+
+## Installation
+
+To install the MVR CLI, you can use the following command:
+
+```bash
+TODO: add installation command
+```
```

Comment:
```suggestion
Download the correct binary file according to your OS and architecture from GitHub release page:
https://github.com/MystenLabs/mvr/releases/latest

After downloading the file, put it on your PATH. Below are examples on how to do it on three popular shells:

### For Zsh
1. Open the `zshrc` file
```bash
nano ~/.zshrc
```
2. Add this line at the end, replacing `/path/to/your/folder` with the actual folder path to the binary:
```bash
 export PATH="/path/to/your/folder:$PATH"
```
3. Save the file and source it to apply changes:
```bash
source ~/.zshrc
```

### For bash
1. Open your `.bashrc` file (or `.bash_profile` on macOS):
```bash
nano ~/.bashrc
```
2. Add this line at the end, replacing `/path/to/your/folder` with the actual folder path to the binary:
```bash
 export PATH="/path/to/your/folder:$PATH"
```
3. Save the file and source it to apply changes:
```bash
source ~/.bashrc
```

### For fish
1. Open your Fish configuration file:
```bash
nano ~/.config/fish/config.fish
```
2. Add this line, replacing `/path/to/your/folder` with your desired path:
```bash
set -Ux fish_user_paths /path/to/your/folder $fish_user_paths
```
3. Fish automatically loads this file, so you don’t need to manually source it.

Now you can call `mvr --help`.
```